Sources/PackageLoading/ToolsVersionParser.swift

```diff
@@ -654,7 +658,7 @@ extension ManifestLoader {
                 regularManifestToolsVersion = try ToolsVersionParser.parse(manifestPath: regularManifest, fileSystem: fileSystem)
             }
             catch {
-                if case ToolsVersionParser.Error.malformedToolsVersionSpecification(.commentMarker(.isMissing)) = error {
+                if let unsupportedError = error as? UnsupportedToolsVersion, unsupportedError.packageToolsVersion == .v3 {
```

Comment:
heh, I think we can do the same catch style here too

Reply:
Good point